src/Batch/Batch/Models/BatchAccountContext.cs

```diff
@@ -110,6 +110,16 @@ public string TagsTable
         /// </summary>
         public int PoolQuota { get; private set; }
 
+        /// <summary>
```

Comment:
We should modify a test to validate proper serialization/deserialization of the new parameters? Also, please add a changelog entry for this.  Note: if you are on an Azure team, please join the azure org (https://aka.ms/azuregithub)

Reply:
Also, it looks like these properties do not exist, do you need to update the SDK?